packages/sdk/viem-account/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@celo/viem-account",
```

Comment:
both to make it clear its for ledger and because if we end up creating more of these each accountToX will require its own dependencies. im thinking we have a package for each. 

```suggestion
  "name": "@celo/viem-account-ledger"
```

Reply:
yeah actually that's an oversight I need to nest this more